MAINTAINERS.md

```diff
@@ -1,10 +1,13 @@
+# OpenAPI Specification Editors
+
 ## Active
 * Darrel Miller [@darrelmiller](https://github.com/darrelmiller)
 * Jeremy Whitlock [@whitlockjc](https://github.com/whitlockjc)
 * Marsh Gardiner [@earth2marsh](https://github.com/earth2marsh)
 * Ron Ratovsky [@webron](https://github.com/webron)
 
-## Provisional
+## Trainee
```

Comment:
What makes Trainee better here? I'm not against it, just trying to understand why this word in particular.

Reply:
Reverted, added separate EDITORS.md